src/github_repo_request_local.py

```diff
@@ -0,0 +1,74 @@
+import subprocess
+from pathlib import Path
+import pandas as pd
+from loguru import logger
+import shutil
+
+
+def count_and_list_test_files(directory):
+    logger.info(f"Processing the directory {directory}")
+    test_files = []
+    # Define a set of file extensions to exclude
+    excluded_extensions = {
```

Comment:
Good, as I've mentioned let's make this configurable at some point soon so that we don't need to edit code to change the exclusion criteria. For example if someone wants to exclude svg files, currently they'd need to format the exclusion correctly here to do so rather than editing a flat text file.

Reply:
Could you please advice if using `argparse` module is a good approach or would it be better if we prompt the user to enter the file extensions to exclude at runtime?

Reply:
- Implemented `argparse` for dynamic exclusion of file extensions, allowing users to specify which files to ignore when counting test files. 
- Logging the excluded file extensions `logger.info(f"Excluded file extensions: {', '.join(args.exclude)}")`
- code has been committed and pushed to the branch